data/src/main/java/com/github/abdallahabdelfattah13/data/repositories/entities/ArticleEntity.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.abdallahabdelfattah13.data.repositories.entities
+
+import java.util.*
+
+data class ArticleEntity(
+    val id: Int,
+    val title: String,
+    val url: String,
+    val publicationDate: Date
+)
```

Comment:
This is not being used. Is that correct?

Reply:
Done